sklearn/utils/multiclass.py

```diff
@@ -167,20 +159,8 @@ def is_label_indicator_matrix(y):
                                     _is_integral_float(labels))
 
 
-def is_sequence_of_sequences(y):
-    """ Check if ``y`` is in the sequence of sequences format (multilabel).
-
-    This format is DEPRECATED.
-
-    Parameters
-    ----------
-    y : sequence or array.
-
-    Returns
-    -------
-    out : bool,
-        Return ``True``, if ``y`` is a sequence of sequences else ``False``.
-    """
+def _is_sequence_of_sequences(y):
```

Comment:
I kept this to make sure we stay consistent of what we previously called `sequence_of_sequence` and [to return it's type as `'unknown'`](https://github.com/scikit-learn/scikit-learn/pull/4919/files#diff-c569d578c4c466c2b310da6a2b06e1caR263)...

We can remove this but we may need to refactor this logic into [these lines](https://github.com/scikit-learn/scikit-learn/pull/4919/files#diff-c569d578c4c466c2b310da6a2b06e1caR271) to make sure all previously supported seq of seqs are considered as `'unknown'`...

Could I do that?